pod.go

```diff
@@ -144,4 +158,16 @@ func (pc *podCollector) collectPod(ch chan<- prometheus.Metric, p v1.Pod) {
 		addGauge(descPodContainerStatusReady, boolFloat64(cs.Ready), cs.Name)
 		addCounter(descPodContainerStatusRestarts, float64(cs.RestartCount), cs.Name)
 	}
+
+	nodeName := p.Spec.NodeName
+	for _, c := range p.Spec.Containers {
+		req := c.Resources.Requests
```

Comment:
As far as I understand resource requests cannot change over a pods lifetime.

```
pod updates may not change fields other than `containers[*].image` or `spec.activeDeadlineSeconds`
```

Therefore it might make sense to put that information into the [PodContainerInfo](https://github.com/ahakanbaba/kube-state-metrics/blob/79761d06617809f78c16a5f129e8267e1e8fe7cd/pod.go#L152). Similar to the PodInfo or NodeInfo metrics.